CMakeLists.txt

```diff
@@ -13,8 +13,19 @@ ADD_LIBRARY(silber-1.0 STATIC
     ClientProcCommunicator.cpp
     ServerProcCommunicator.cpp)
 
-install(TARGETS silber-1.0 DESTINATION lib)
-install(FILES Message.hpp DESTINATION include )
+# install library if it is needed by multiple targets
+install(TARGETS silber-1.0
+EXPORT silber-1.0-config
+ DESTINATION lib)
+
```

Comment:
depending on the installation path:
export LIBRARY_PATH="$LIBRARY_PATH:/usr/local/lib"